.github/workflows/docker.yml

```diff
@@ -45,3 +45,20 @@ jobs:
         run: curl --fail -X POST https://staging.semgrep.dev/api/admin/update-docker
       - name: update semgrep.dev
         run: curl --fail -X POST https://semgrep.dev/api/admin/update-docker
+
+      # Extend the semgrep image, changing the entry point to bash and
+      # adding some utilities. This image is meant for internal uses
+      # such as benchmarks.
+      - name: Build the extended semgrep-dev Docker image
+        run: |
+          docker build \
+            -f dockerfiles/semgrep-dev.Dockerfile \
+            -t returntocorp/semgrep-dev .
+      - name: Push develop image if on develop branch
+        if: ${{ github.event_name == 'push'}}
+        uses: docker/build-push-action@v1
+        with:
+          username: ${{ secrets.DOCKER_USERNAME }}
+          password: ${{ secrets.DOCKER_PASSWORD }}
+          repository: returntocorp/semgrep-dev
+          tags: develop
```

Comment:
The critical piece is the `repository` line here, so that we don't clobber the other image.